### DISCUSSION TITLE
Не получается удалить zapret

### DISCUSSION THREAD

**Discussion by franerky** [bol-van/zapret#953] [Q&A]
﻿Здравствуйте!
﻿У меня не получается удалить программу zapret через service_remote. 
﻿Всегда выдаёт ошибку: Restarted with admin rights
The service name is invalid.

More help is available by typing NET HELPMSG 2185.

[SC] OpenService FAILED 1060:

The specified service does not exist as an installed service.

The service name is invalid.

More help is available by typing NET HELPMSG 2185.

[SC] OpenService FAILED 1060:

The specified service does not exist as an installed service.

The service name is invalid.

More help is available by typing NET HELPMSG 2185.

[SC] OpenService FAILED 1060:

The specified service does not exist as an installed service.

**Comment by bol-van** [comment]
В варианте из коробки все батники для создания служб закомментированы REM.
Это сделано специально, чтобы нубы не жали на все подряд и не задавали потом подобные вопросы.
У вас либо никогда не было службы, либо вы точно должны знать как ее удалять.
Либо у вас сборка, а по сборкам все вопросы к сборщикам
